osmnx/downloader.py

```diff
@@ -266,9 +266,18 @@ def _get_host_by_name(host):
     ip_address : string
         resolved IP address
     """
-    dns_url = f"https://8.8.8.8/resolve?name={host}"
-    response = requests.get(dns_url)
-    data = response.json()
+    if settings.dns_http_url is None:
+        return host
+
+    dns_url = settings.dns_http_url % f'name={host}'
```

Comment:
This appears to be the error. I fixed it in: https://github.com/a-gerhard/osmnx/pull/1